### PR TITLE
DOMA-3443 optimize reference localizations

### PR DESCRIPTION
**Pull request by akarjakin**

### CODE REVIEW

File: apps/condo/domains/meter/gql.js [hunk 1]
```diff
@@ -10,13 +10,13 @@ const { generateGqlQueries } = require('@condo/domains/common/utils/codegenerati
...
+const METER_RESOURCE_FIELDS = `{ name nameNonLocalized measure measureNonLocalized ${COMMON_FIELDS} }`
 const MeterResource = generateGqlQueries('MeterResource', METER_RESOURCE_FIELDS)
 
-const METER_READING_SOURCE_FIELDS = `{ type name ${COMMON_FIELDS} }`
+const METER_READING_SOURCE_FIELDS = `{ type name nameNonLocalized ${COMMON_FIELDS} }`
 const MeterReadingSource = generateGqlQueries('MeterReadingSource', METER_READING_SOURCE_FIELDS)
 
-const METER_FIELDS = `{ number numberOfTariffs installationDate commissioningDate verificationDate nextVerificationDate controlReadingsDate sealingDate accountNumber organization { id } property { id address addressMeta { ${ADDRESS_META_SUBFIELDS_QUERY_LIST} } } unitName unitType place resource { id name measure } ${COMMON_FIELDS} }`
+const METER_FIELDS = `{ number numberOfTariffs installationDate commissioningDate verificationDate nextVerificationDate controlReadingsDate sealingDate accountNumber organization { id } property { id address addressMeta { ${ADDRESS_META_SUBFIELDS_QUERY_LIST} } } unitName unitType place resource { id name nameNonLocalized measure measureNonLocalized } ${COMMON_FIELDS} }`
```
Comment (pahaz) [inline apps/condo/domains/meter/gql.js:19]: It’s better to reuse meter reading resource and source field

**Review by pahaz — APPROVED** [review]

**Review by ekabardinsky — APPROVED** [review]

File: apps/condo/domains/common/utils/localesLoader.js [hunk 2]
```diff
@@ -34,6 +34,13 @@ const getAvailableLocales = () => {
     return Object.keys(translations)
 }
 
+const getLocalized = (lang, key) => {
+    const translations = getTranslations(lang)
+
+    return translations[key]
```
Comment (AntonAL) [inline apps/condo/domains/common/utils/localesLoader.js:40]: What will happen of provided key is missing?

File: packages/@core.keystone/fields/LocalizedText/Implementation.js [hunk 3]
```diff
@@ -13,6 +13,10 @@ class LocalizedText extends Text.implementation {
         }
     }
 
+    gqlOutputFields () {
```
Comment (AntonAL) [inline packages/@core.keystone/fields/LocalizedText/Implementation.js:16]: Will it not produce redundant migrations for fields in Keystone?

File: apps/condo/domains/meter/gql.js (hunk 1 above)
Comment (akarjakin) [reply apps/condo/domains/meter/gql.js]: done

**Pull request merged**